### DISCUSSION TITLE
ez filter hz error

### DISCUSSION THREAD

**Discussion by AlterMannFliegt** [iNavFlight/inav#9523] [Q&A]
INAV 7.0.0 RC3 and SpeedyBee F405 Wing:
Invalid setting:  ez_filter_hz  and  ez_axis_ratio
default for both terms is 110, but it is resetting to 0 after a while
After autotune the FeedForward raised to 255 on Pitch and 110 on Roll.   WHY ???

**Comment by dcan999** [comment]
I have the ez_filter_hz invalid setting on Matek 405TE RC3 as well.

**Comment by MrD-RC** [comment]
It’s a known issue when updating from an earlier RC. Just set those parameters to the default value in the CLI.

**Reply by AlterMannFliegt** [reply]
I had a new FC and the values reset themselves from 110 to 0. It was no update from a previous RC ☝️
